Core/Core/CourseSync/Common/ViewModels/OfflineListCellViewModel.swift

```diff
@@ -133,7 +133,29 @@ class OfflineListCellViewModel: ObservableObject {
                 progressText = String(localized: "Downloading", bundle: .core)
             }
         }
-        return titleText + "," + selectionText + "," + collapseText + "," + progressText + ","
```

Comment:
Is removing the final comma intentional in the rewritten code?
If so, there is a String method I've added recently: `nilIfEmpty` which allows for something like this:
`[titleText.nilIfEmpty, selectionText.nilIfEmpty, ...].compactMap{$0}.join(separator: ", ")`

Reply:
It is intentional because it looked weird. I'd really like to merge these changes so I won't modify it again.